synapse/events/spamcheck.py

```diff
@@ -306,7 +376,7 @@ async def check_event_for_spam(
                 elif res is True:
                     # This spam-checker rejects the event with deprecated
                     # return value `True`
-                    return Codes.FORBIDDEN
+                    return (Codes.FORBIDDEN, {})
```

Comment:
Ah, good point.

Still, more comfortable with specifying both the subclass and the superclass, since they have different treatment, if that's ok with you.

Reply:
Sure :)